chart/templates/_helpers.tpl

```diff
@@ -8,6 +8,11 @@ app: shell
 release: {{ .Release.Name }}
 {{- end }}
 
+{{- define "elastic.release_labels" }}
```

Comment:
The prefix in these template definitions should match the chart name, so in this case it should be `drupal`, not `elastic` (by the way, Elastic is the company, Elasticsearch is the software).

Reply:
https://github.com/wunderio/drupal-project-k8s/pull/74/commits/041607b1c4bfdd62a4aba1f4f949d6c4fdf7130c  
Inserted common label set (app,release) for shell, elasticsearch resources. For selectors to differentiate between these, I added third label `service` to mark which accompanying service it is, i.e. `service: shell`